qemu/tests/hpt_miscellaneous.py

```diff
@@ -38,6 +41,18 @@ def verify_hpt(test, params, session, hpt_size):
                           % (get_hpt_value, hpt_size))
 
 
+def check_mem_increase(session, params, orig_mem, in_mem):
```

Comment:
`def check_mem_increase(session, params, orig_mem, in_mem): `- >` def check_mem_increase(session, params, orig_mem, increase_mem):`

Reply:
Updated it from in_mem to increase_mem.